src/core/obj.js

```diff
@@ -703,12 +703,18 @@ var Catalog = (function CatalogClosure() {
               'app.launchURL',
               'window.open'
             ];
-            var regex = new RegExp('^(?:' + URL_OPEN_METHODS.join('|') + ')' +
-                                   '\\((?:\'|\")(\\S+)(?:\'|\")(?:,|\\))');
+            var regex = new RegExp('^\\s*(' + URL_OPEN_METHODS.join('|') + ')' +
+                                   '\\((?:\'|\")(\\S+)(?:\'|\")' +
```

Comment:
let's replace `\\S+` to `[^\'\:]*` -- we don't parse or handle JS string properly anyway -- this will at least be to that eager to get more of the content.